### PR TITLE
fix: set the correct path to use pvc volume

### PR DESCRIPTION
**Pull request by Henrrypg**
# Description

This PR fixes the path to use in db backup to /data which is mounted by PVC. 

Also changes the cronjob to just run the docker entrypoint with arg and changes the docker image.

# Test

I was testing on [yamato-dunkirk](https://argocd.services.stage.edunext-shipyard.net/applications/dunkerque-openedx?operation=false&resource=)

### CODE REVIEW

File: drydock_backups/docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      mysqldump -u $MYSQL_ROOT_USERNAME -h $MYSQL_HOST -P $MYSQL_PORT --password=$MYSQL_ROOT_PASSWORD \
+            --all-databases --single-transaction --flush-logs | gzip > $FILENAME
+elif [ "$1" = 'mongo' ]; then
+      if [[ -z "${MONGODB_USERNAME}" ]]; then
+            mongodump \
+            --host $MONGODB_HOST:$MONGODB_PORT \
+            -d $MONGODB_DATABASES --gzip \
+            --archive=$FILENAME
+      else
+            mongodump --username $MONGODB_USERNAME --password $MONGODB_PASSWORD --authenticationDatabase=admin \
```
Comment (MoisesGSalas) [inline drydock_backups/docker-entrypoint.sh:18]: I think we can build a [uri](https://www.mongodb.com/docs/manual/reference/connection-string/) instead?

we can add something like

```bash
MONGODB_URI="mongodb://"
MONGODB_PORT=${MONGODB_PORT:-27017}
MONGODB_HOST=${MONGODB_HOST:-mongodb}
MONGODB_AUTHDB=${MONGODB_AUTHDB:-admin}

if [ -n "${MONGODB_USERNAME}" ] && [ -n "${MONGODB_PASSWORD}" ]; then
    MONGODB_URI+="${MONGODB_USERNAME}:${MONGODB_PASSWORD}@"
fi

MONGODB_URI+="${MONGODB_HOST}:${MONGODB_PORT}/${MONGODB_DATABASES}?authSource=${MONGODB_AUTHDB}"
```

at the start of the file and do something like:

```bash
elif [ "$1" = 'mongo' ]; then
      
    mongodump --uri="${MONGODB_URI}" --gzip --archive=$FILENAME
else
```

I think we shouldn't set the `MONGODB_DATABASES` value by default, at the moment we only are dumping the DB used by edxapp and if forum uses a different DB then it's not included in the backup. If we leave the database blank (e.g. `mongodb://root:secure-password@mongodb:27017/?authSource=admin`) it should dump all the DBs.

File: drydock_backups/patches/drydock-multipurpose-jobs
Comment (MoisesGSalas) [inline drydock_backups/patches/drydock-multipurpose-jobs:1]: I think this value should be empty by default:

https://github.com/eduNEXT/drydock/blob/52a518594ae84e49730570fff576cde9206d3e68/drydock_backups/patches/drydock-multipurpose-jobs#L102-L103

`MONGODB_DATABASE` is a tutor variable that defaults to `openedx`. Sometimes we use a different database for edxapp and forum, if we use this value as is then we only make a backup for edxapp but not forum.

Comment (Henrrypg) [reply drydock_backups/patches/drydock-multipurpose-jobs:1]: @MoisesGSalas I just add a new plugin variable `BACKUP_MONGODB_DATABASE` empty at default to keep the way to set it from the config.yml

File: drydock_backups/plugin.py [hunk 2]
```diff
@@ -25,7 +25,8 @@
         "MYSQL_USERNAME": '{{ MYSQL_ROOT_USERNAME }}',
         "MYSQL_PASSWORD": '{{ MYSQL_ROOT_PASSWORD }}',
         "MONGO_PASSWORD": '{{ MONGODB_PASSWORD }}',
-        "MONGO_USERNAME": '{{ MONGODB_USERNAME }}'
+        "MONGO_USERNAME": '{{ MONGODB_USERNAME }}',
+        "MONGODB_DATABASE": None
```
Comment (MoisesGSalas) [inline drydock_backups/plugin.py:29]: Empty string would be better.

**Review by MoisesGSalas — APPROVED** [review]

**Pull request merged**